modules/host/contractmanager/sector.go

```diff
@@ -147,6 +147,10 @@ func (cm *ContractManager) ReadSector(root crypto.Hash) ([]byte, error) {
 		cm.log.Critical("Unable to load storage folder despite having sector metadata")
 		return nil, ErrSectorNotFound
 	}
+	if atomic.LoadUint64(&sf.atmoicUnavailable) == 1 {
```

Comment:
I don't know how you can live without autocomplete on identifiers lol. It probably saves me thousands of keystrokes per day.